src/main.rs

```diff
@@ -481,11 +481,17 @@ async fn run() -> Result<(), S2CliError> {
                 }
                 None => (None, None),
             };
-            account_service
+            let BasinInfo { state, .. } = account_service
                 .create_basin(basin.into(), storage_class, retention_policy)
                 .await?;
 
-            eprintln!("{}", "✓ Basin created".green().bold());
+            let message = match state {
+                streamstore::types::BasinState::Creating => {
+                    "✓ Basin creation requested".yellow().bold()
+                }
+                _ => "✓ Basin created".green().bold(),
+            };
+            eprintln!("{message}");
         }
```

Comment:
this makes sense, deletion/unspecified state cant really happen in this case so I guess its ok to hide it